metadata-service/factories/src/main/java/com/linkedin/gms/factory/entity/update/indices/UpdateIndicesServiceFactory.java

```diff
@@ -35,9 +40,27 @@ public class UpdateIndicesServiceFactory {
   @Value("${elasticsearch.search.graph.graphStatusEnabled}")
   private boolean graphStatusEnabled;
 
+  /** Creates a collection of UpdateIndicesStrategy instances based on Spring beans. */
+  private Collection<UpdateIndicesStrategy> createStrategies(
+      @Qualifier("updateIndicesV2Strategy") @Nullable UpdateIndicesStrategy v2Strategy,
+      @Qualifier("updateIndicesV3Strategy") @Nullable UpdateIndicesStrategy v3Strategy) {
+
+    Collection<UpdateIndicesStrategy> strategies = new ArrayList<>();
+
+    if (v2Strategy != null) {
+      strategies.add(v2Strategy);
+    }
+
+    if (v3Strategy != null) {
+      strategies.add(v3Strategy);
+    }
```

Comment:
Probably log when strategies are added. If v2 and v3 are left enabled during the migration and an operator forgets to turn v2 off would be helpful to get some logs to know that we are doing dual writes.